k8s/ebs-terraform/variables.tf

```diff
@@ -3,7 +3,8 @@ variable "aws_region" {}
 variable "aws_availability_zone" {}
 
 variable "default_tags" {
-  type = "map"
+  type = map
+  description = "change KubernetesCluster to fit your cluster name"
 
   default = {
     Name              = "taas-daemon-datadir-volume"
```

Comment:
(creating here to get a discussion thread)

Do we need other fixes from https://github.com/testground/infra/pull/70/files ?

the `ZONE` requirements and fixing the GitHub URLs were mandatory if I recall correctly.

Reply:
I believe these are all the changes needed to successfully boot up a cluster. At least this is how i got it working.
Never changed any `ZONE` variable, but I know that it needs to be set as an env var before you can create a cluster.

Reply:
Got it, let's merge as is then and revisit if someone else raises the issue.

In the spirit of leaving things a little better than we found them, can we at least fix the URLs in k8s/testground-infra/Chart.yaml and k8s/testground-infra/charts/testground-dashboards/Chart.yaml ?